packages/xstate-inspect/src/browser.ts

```diff
@@ -146,13 +146,13 @@ export function inspect(options?: InspectorOptions): Inspector | undefined {
     stringify(value, options?.serialize);
 
   devTools.onRegister((service) => {
-    const state = service.state || service.initialState;
+    const state = service.getSnapshot() || service.initialState;
```

Comment:
If I read things correctly the fallback here won't ever be used (or it will be used incorrectly). Dev tools are attached when the interpreter is already in its running state and thus `getSnapshot` should always return the current internal state:
https://github.com/statelyai/xstate/blob/f88639cf324d4f2e0b8b02ab482e5ec9ba6e0aaf/packages/core/src/interpreter.ts#L374

```suggestion
    const state = service.getSnapshot()
```